docs/visual-testing/integrations/cypress.md

```diff
@@ -188,7 +189,8 @@ Below are the environment variables available for the Sauce Visual for Cypress p
 | `SAUCE_ACCESS_KEY`      | required | Your Sauce Labs access key. You can get this from the header of app.saucelabs.com                                                                                                                                                |
 | `SAUCE_REGION`          |          | The region you'd like to run your Visual tests in. Defaults to `us-west-1` if not supplied. Can be one of the following: <br/> `'eu-central-1'`, `'us-west-1'` or `'us-east-4'`                                                  |
 | `SAUCE_VISUAL_BUILD_NAME`      |          | The name you would like to appear in the Sauce Visual dashboard. Defaults to 'Cypress Visual Testing'                                                                                                                                   |
-| `SAUCE_VISUAL_BRANCH_NAME`     |          | The branch name or tag you would like to associate this build with. We recommend using your current VCS branch in CI.                                                                                                            |
+| `SAUCE_VISUAL_BRANCH_NAME`     |          | The branch name you would like to associate this build with. We recommend using your current VCS branch in CI.                                                                                                            |
+| `SAUCE_VISUAL_DEFAULT_BRANCH_NAME`     |          | The main branch name you would like to associate this build with. Usually `main` or `master` or alternatively the branch name your current branch was derived from.                                                                                                            |
```

Comment:
> The main branch name you would like to associate this build with.

This doesn't really explain the functionality / behavior behind this option. Do you think we should link to the branching behavior mechanics so it's more clear?

Reply:
Changed this for cypress and storybook. For all the others this will be aligned when we add env variables for them.